command/operator_debug.go

```diff
@@ -48,6 +47,7 @@ type OperatorDebugCommand struct {
 	manifest      []string
 	ctx           context.Context
 	cancel        context.CancelFunc
+	opts          *api.QueryOptions
```

Comment:
Note for reviewers: I'm not wild about having it stuck in the `OperatorDebugCommand` like this, but the command keeps around a lot of state here and untangling all that was more of a refactor than I wanted to do in this PR. I can come back and do some refactoring later.

Reply:
I think this looks OK. _Something_ would need to be passed around regardless, and yeah, there's a lot going on in this command 😄